fwoa-tools/src/binaryConverter.unit.test.ts

```diff
@@ -0,0 +1,267 @@
+/*
```

Comment:
Oh for these test files, you don't need to include `unit` in the name. It could just be `<fileName>.test.ts`. Yanyu made a change in which the word `unit` doesn't have to be included anymore.